web/libs/core/src/pages/AccountSettings/sections/PersonalAccessToken.tsx

```diff
@@ -71,7 +70,8 @@ export const PersonalAccessToken = () => {
             <TextArea
               name="example-curl"
               readOnly
-              className={clsx(styles.input, styles.textarea, "w-full min-h-[90px]")}
+              className={styles.textarea}
+              rawClassName={styles.textarea}
```

Comment:
that definitely smells. I don't like and understand the idea of `rawClassName` especially given that we have `mix` already. if that's just a temporary solution would be nice to this explicitely described.

Reply:
mix will still add prefixes. it's temp but will only go away when we rewrite bem here